<launch>

<param name="yaw_rate/kp" value="20.0"/>
<param name="yaw_rate/ki" value="1"/>
<param name="yaw_rate/kd" value="5"/>



<node pkg="kingfisher_node" type="kingfisher_controller" name="kingfisher_controller" output="screen"/>

</launch>
